### PR TITLE
Update BCD keys for SVG attributes

### PR DESCRIPTION
**Pull request by queengooborg**
This PR performs a number of updates to the SVG attributes pages to clean up the pages:
- **Remove mention of global attributes on individual elements pages** (https://github.com/mdn/content/pull/32383)
- **Remove landing pages for attribute categories** (https://github.com/mdn/content/pull/32477)
- **Update BCD keys** (Only part remaining)

Note: review and merge along with https://github.com/mdn/browser-compat-data/pull/22267.

Follow-up: remove all use of the `SVGAttr` macro.

### CODE REVIEW

**Review by chrisdavidmills — APPROVED** [review]
I'm not really able to check this all in detail, but the pages I've looked at LGTM; nice work @queengooborg.

**Comment by queengooborg** [comment]
Thank you, @chrisdavidmills!

Would you also be down to review/approve the BCD PR by chance?

**Comment by chrisdavidmills** [comment]
> Thank you, @chrisdavidmills!
> 
> Would you also be down to review/approve the BCD PR by chance?

I am not, no. I can do, if no one grabs it.

File: files/en-us/web/svg/element/a/index.md [hunk 1]
```diff
@@ -95,23 +95,6 @@ svg|a:active {
...
-  - : [Global event attributes](/en-US/docs/Web/SVG/Attribute/Events#global_event_attributes), [Document element event attributes](/en-US/docs/Web/SVG/Attribute/Events#document_element_event_attributes), [Graphical event attributes](/en-US/docs/Web/SVG/Attribute/Events#graphical_event_attributes)
-- [Presentation Attributes](/en-US/docs/Web/SVG/Attribute/Presentation)
-  - : Most notably: {{SVGAttr('clip-path')}}, {{SVGAttr('clip-rule')}}, {{SVGAttr('color')}}, {{SVGAttr('color-interpolation')}}, {{SVGAttr('color-rendering')}}, {{SVGAttr('cursor')}}, {{SVGAttr('display')}}, {{SVGAttr('fill')}}, {{SVGAttr('fill-opacity')}}, {{SVGAttr('fill-rule')}}, {{SVGAttr('filter')}}, {{SVGAttr('mask')}}, {{SVGAttr('opacity')}}, {{SVGAttr('pointer-events')}}, {{SVGAttr('shape-rendering')}}, {{SVGAttr('stroke')}}, {{SVGAttr('stroke-dasharray')}}, {{SVGAttr('stroke-dashoffset')}}, {{SVGAttr('stroke-linecap')}}, {{SVGAttr('stroke-linejoin')}}, {{SVGAttr('stroke-miterlimit')}}, {{SVGAttr('stroke-opacity')}}, {{SVGAttr('stroke-width')}}, {{SVGAttr("transform")}}, {{SVGAttr('vector-effect')}}, {{SVGAttr('visibility')}}
-- XLink Attributes
-  - : Most notably: {{SVGAttr("xlink:title")}}
```
Comment (estelle) [inline files/en-us/web/svg/element/a/index.md:111]: when there is a "most notably" global attribute on an SVG page, we should ensure that it is somewhere on the page. Add a See Also to all the pages htat have a "most notably" global attribute if that attribute is not already in the context of page.

File: files/en-us/web/svg/element/animate/index.md [hunk 2]
```diff
@@ -35,28 +35,6 @@ svg {
 
 {{EmbedLiveSample('Example', 150, '100%')}}
 
-## Attributes
-
```
Comment (estelle) [inline files/en-us/web/svg/element/animate/index.md:39]: ensure the following links are in the page; if not in the content, within a "see also"

- {{SVGAttr("attributeName")}}
- {{SVGAttr("additive")}}
- {{SVGAttr("accumulate")}}
and possibly these:
- {{SVGAttr("onbegin")}}
- {{SVGAttr("onend")}}
- {{SVGAttr("onrepeat")}}

**Review by estelle — DISMISSED** [review]
I made a few comments. This PR is too big! Would it be possible to break it into 3: 
1. the updating of browser compat in the front matter
2. removal of global attributes (while ensuring the links to those under "most notably" are included in their relevant pages
3. removal of the attribute category landing pages

I spot checked for #1, and think it's ok.
Changes are required for #2.
I did not look at category 3.

**Comment by queengooborg** [comment]
I can try splitting it into individual PRs, but I'm afraid there's a lot of crossover between the changes and they must be merged in order to prevent issues.

I'd probably recommend reviewing this PR per commit instead of all at once?

**Comment by estelle** [comment]
Review #1: assuming BCD was updated, there are no visible issues here.

**Review by estelle — COMMENTED** [review]
Review part 2: Remove landing pages for attribute categories

A few minor issues in this section.

I'll review part 3 after the pages are updated to ensure the important attributes are not omitted from the element pages.

File: files/en-us/web/svg/attribute/presentation/index.md [hunk 3]
```diff
@@ -1,274 +0,0 @@
...
-- [marker-end](#marker-end)
-- [marker-mid](#marker-mid)
-- [marker-start](#marker-start)
-- [mask](#mask)
-- [opacity](#opacity)
-- [overflow](#overflow)
-- [pointer-events](#pointer-events)
-- [shape-rendering](#shape-rendering)
-- [solid-color](#solid-color)
-- [solid-opacity](#solid-opacity)
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/svg/attribute/presentation/index.md:56]: missing from the list on the main attribute page.

File: files/en-us/web/svg/attribute/presentation/index.md [hunk 4]
```diff
@@ -1,274 +0,0 @@
...
-- [clip](#clip)
-- [clip-path](#clip-path)
-- [clip-rule](#clip-rule)
-- [color](#color)
-- [color-interpolation](#color-interpolation)
-- [color-interpolation-filters](#color-interpolation-filters)
-- [color-profile](#color-profile)
-- [color-rendering](#color-rendering)
-- [cursor](#cursor)
-- [d](#d)
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/svg/attribute/presentation/index.md:23]: missing from the list on the main attribute page.

File: files/en-us/web/svg/attribute/presentation/index.md [hunk 5]
```diff
@@ -1,274 +0,0 @@
...
-- [stroke-dashoffset](#stroke-dashoffset)
-- [stroke-linecap](#stroke-linecap)
-- [stroke-linejoin](#stroke-linejoin)
-- [stroke-miterlimit](#stroke-miterlimit)
-- [stroke-opacity](#stroke-opacity)
-- [stroke-width](#stroke-width)
-- [text-anchor](#text-anchor)
-- [text-decoration](#text-decoration)
-- [text-rendering](#text-rendering)
-- [transform](#transform)
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/svg/attribute/presentation/index.md:70]: this doesn't matter ;)

**Comment by queengooborg** [comment]
This PR is now ready for re-review and merging, along with the BCD PR!

**Comment by queengooborg** [comment]
The BCD PR has been approved and merged, so this will be merged once the next BCD release is out!

**Review by estelle — APPROVED** [review]

**Pull request merged**